docs/build/pulse.ipynb

```diff
@@ -770,6 +776,59 @@
     "    pulse.shift_phase(np.pi, channel)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `shift_frequency`\n",
+    "\n",
+    "The `shift_frequency` instruction shifts the `frequency` of a pulse `channel`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d0 = pulse.DriveChannel(0)\n",
+    "\n",
+    "with pulse.build() as pulse_prog:\n",
+    "    pulse.shift_frequency(1e9, d0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `set_phase`\n",
+    "\n",
+    "The `set_phase` instruction sets the phase of a pulse channel."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d0 = pulse.DriveChannel(0)\n",
+    "\n",
+    "with pulse.build() as pulse_prog:\n",
+    "    pulse.set_phase(math.pi, d0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"note\" title=\"Note\">\n",
+    "The `shift_frequency`, `shift_phase`, `set_frequency`, and `set_phase` instructions are not only changing the frequency of following pulses - they also change the frame of reference of the channel. Because a qubit oscillates at the frequency of its transition, the controller needs to sync with its oscillation; otherwise, unwanted Z drive is continuously applied. Usually the frame is matched with the frequency of the drive, and drive matches with the frequency of the transition; the Z drive is eliminated when the qubit frequency is calibrated properly. When you apply the `shift_frequency` instruction, it changes the drive frequency and impacts the frame; in other words, it accumulates the phase (Z) as a function of shifted frequency and duration of the program. Specifically, when you shift the frequency by `df` and spend `dt` on that frame, the qubit may experience a phase rotation of `df * dt`. The programmer needs to take this into account to control their qubits precisely. \n",
```

Comment:
```suggestion
    "The `shift_frequency` and `set_frequency` instructions are not only changing the frequency of following pulses - they also change the frame of reference of the channel. Because a qubit oscillates at the frequency of its transition, the controller needs to sync with its oscillation; otherwise, unwanted Z drive is continuously applied. Usually the frame is matched with the frequency of the drive, and drive matches with the frequency of the transition; the Z drive is eliminated when the qubit frequency is calibrated properly. When you apply the `shift_frequency` instruction, it changes the drive frequency and impacts the frame; in other words, it accumulates the phase (Z) as a function of shifted frequency and duration of the program. Specifically, when you shift the frequency by `df` and spend `dt` on that frame, the qubit may experience a phase rotation of `df * dt`. The programmer needs to take this into account to control their qubits precisely. \n",
```